src/Entity/Transformer.php

```diff
@@ -150,6 +150,7 @@ public function create(PageInterface $page, ?SnapshotInterface $snapshot = null)
             /**
              * @var PageContent $content
              */
+            // @phpstan-ignore-next-line
```

Comment:
Please revert this, I'll merge with the phpstan error.
We have an issue for this https://github.com/sonata-project/SonataPageBundle/issues/1728